engines/sci/graphics/font.cpp

```diff
@@ -38,7 +38,9 @@ GfxFontFromResource::GfxFontFromResource(ResourceManager *resMan, GfxScreen *scr
 
 	_resource = resMan->findResource(ResourceId(kResourceTypeFont, resourceId), true);
 	if (!_resource) {
-		error("font resource %d not found", resourceId);
+		// The Lighthouse demo does not contain any fonts
```

Comment:
Some (all?) SCI3 interpreters (and it seems SCI2.1 as well) contain a fallback font embedded in the interpreter, which they will use if given the font number -1 (65535). A backtrace shows that the bad font lookup comes from the constructor to GfxText32 and not from the scripts... if we absolutely must have a font, would it not be better to implement this embedded font and use it?